tutorials/math/Bessel.py

```diff
@@ -0,0 +1,181 @@
+## \file
+## \ingroup tutorial_math
+## \notebook
+## Show the different kinds of Bessel functions available in ROOT
+## To execute the macro type in:
+##
+## ~~~{.cpp}
+## root[0] .x Bessel.C
+## ~~~
+##
+## It will create one canvas with the representation
+## of the  cylindrical and spherical Bessel functions
+## regular and modified
+##
+## Based on Bessel.C by Magdalena Slawinska
+##
+## \macro_image
+## \macro_code
+##
+## \author Juan Fernando Jaramillo Botero
+
+from ROOT import TCanvas, TF1, gSystem, gPad, TLegend, TPaveLabel, kBlack
+
+
+gSystem.Load("libMathMore")
+
+DistCanvas = TCanvas("DistCanvas", "Bessel functions example", 10, 10, 800, 600)
+DistCanvas.SetFillColor(17)
+DistCanvas.Divide(2, 2)
+DistCanvas.cd(1)
+gPad.SetGrid()
+gPad.SetFrameFillColor(19)
+leg = TLegend(0.75, 0.7, 0.89, 0.89)
+
+# Drawing the set of Bessel J functions
+#
+# n is the number of functions in each pad
+n = 5
+JBessel = []
+for nu in range(n):
+    _jbessel = TF1("J_0", "ROOT::Math::cyl_bessel_j([0],x)", 0, 10)
```

Comment:
why this name "_jbessel"?

Reply:
Can we with the new TFormula avoid params called "[0]"?

Reply:
Do you refer to use TFunction instead TF1? I don't understand. TFunction as a I saw receive analitical functions, but in this case the bessel function is predefined in ROOT::Math::cyl_bessel_